centaur/src/main/resources/azureBlobTestCases/azure_blob_storage_read/azure_blob_storage_read.wdl

```diff
@@ -0,0 +1,12 @@
+version 1.0
+
+workflow azure_blob_storage_read {
+
+  input {
+    File file1
+  }
+
+  output {
+    String s1 = read_string(file1)
```

Comment:
It looks like running this workflow doesn't result in any files being written to the blob container, is that right? Can we use a slightly more complicated workflow that will use the `Local` filesystem rather than TES, and execute a task?

Reply:
This does result in a log being written to the storage container with the supplied options. I can work on a different wdl though to run a task locally if you think thats a clearer test!

Reply:
It seems better to exercise all parts of the system. I know I'm late to the party here, if it ends up not being an easy change I'm fine calling it out of scope.

Reply:
This ended up being a more difficult change because the `Local` backend doesn't easily support working on filesystems other than the local one. For now I think this may need to wait.